DESCRIPTION

```diff
@@ -16,11 +16,12 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.2
 Depends: 
     R (>= 3.2.0)
+Additional_repositories:
+    https://MarkMc1089.github.io/drat
```

Comment:
If deciding to use `{drat}`, would want to add a `drat` repo under nhsbsa-data-analytics. Let me know if you want this. For reference, instructions are [here](https://cran.r-project.org/web/packages/drat/vignettes/DratStepByStep.html).

Reply:
Think we can use Remotes instead of this method but everything else looks good to merge - the name changed from nhsbsR -> nhsbsaR mid development! Glad you are contributing!